README.rst

```diff
@@ -1,17 +1,17 @@
 Overview
 +++++++++++++++++++++
 
-- HSR-Bのコントローラ・センサなどの実機・Gazebo共通設定ファイル管理パッケージを管理する。
+- Manages the common configuration files management package for the HSR-B controllers, sensors, and such for the actual robot and Gazebo.
```

Comment:
I think the Japanese explanation looks wordy :(
How about 
```
 Manages the common configuration files for the HSR-B controllers, sensors, and such for the actual robot and Gazebo.
```

Reply:
Ah, yes.  I see what you did there.
This looks good.
I will edit the text and re-push it.